contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,168 @@
+---
+date: 2022-11-02
+title: "Array 1.41.0: Improving performance by up to 400%"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send every two weeks!
+
+## 1.41.0 update guide for self-hosted users
+Due to changes in this update, it is important check `alias` usage before [upgrading PostHog](/docs/runbook/upgrading-posthog) to 1.41.0 on a self-hosted instance. For example, assuming `email` is used as the identified user id, then: 
+
+```
+identify(email)                                   # in the frontend
+alias(backend_unique_id, email)      # in the backend - THIS WILL NOT WORK
+alias(email, backend_unique_id2)    # in the backend - the right order
```

Comment:
```suggestion
identify(email)                     # in the frontend
alias(email, backend_unique_id)     # in the backend - this works
alias(backend_unique_id, email)     # in the backend - THIS WILL NOT WORK
```